### DISCUSSION TITLE
Watch ArgoApps status in GUI while using appsets

### DISCUSSION THREAD

**Discussion by twanbeeren** [argoproj/argo-cd#11878] [Ideas]
The question is very simple. When I create my appset and generate a new instance, I would like see the status of my application in the GUI. Just to enhance the UX. 

![image](https://user-images.githubusercontent.com/36261352/210586340-19f55a78-462a-498a-8ba0-82219c2bbcb3.png)

Now I have to open the application and see whether its failing or not. Most of the time I generate multiple applications through an appset at once. 

Can I see the status of the apps from this GUI. And if not, why not? It would be a great feature to see if the apps are healthy and in sync

**Comment by morey-tech** [comment]
Hey @twanbeeren! This feature was removed based on the discussion in https://github.com/argoproj/argo-cd/issues/3781 (it's based around App of Apps but should apply to ApplicationSets too). However, you should be able to add it back by [adding a health check for the Application resource](https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/1.7-1.8/#health-assessment-of-argoprojioapplication-crd-has-been-removed), but it's important to understand the implications of doing so (as mentioned in the issue linked).
